installer.php

```diff
@@ -166,26 +166,26 @@
     /*
      * 3.6.0
      */
-    "suggest_run_update"        => "Command unavailable. Please run app:update to download and upgrade to the latest available version: %s.",
-    "suggest_run_upgrade"       => "An upgrade is already in progress. Please run app:upgrade instead.",
-    "failed_to_download"        => "Failed to download file: :message",
-    "backup_failed_abort"       => "Backup failed. Aborting update.",
-    "update_failed"             => "System update failed: :message.",
-    "command_failed"            => "Command :command, failed with output: :message.",
-    "already_latest_version"    => "You are already running the latest version (:version).",
-    "update_not_supported"      => "Automatic update to latest version (:version) is not supported.",
-    "current_and_new_version"   => "You have version :current_version. Latest version available is :new_version.",
-    "update_to_version"         => "You have version :current_version. Would you like to update to :new_version?",
-    "downloading_version"       => "Downloading version :version to :path...",
-    "checksum_not_matched"      => "The downloaded file failed checksum verification.",
-
-    "check_readiness"           => "Checking if the application is able to upgrade...",
-    "read_only"                 => "The filesystem at \":path\" is read only. Updates cannot be applied against a read only file system.",
-    "file_ownership"            => "File \":path\" is owned by \":owner\" but PHP is running as \":actual\". The file owner and PHP user must be the same during an update for all application files.",
-    "minimum_disk_space"        => "Directory \":path\" has insufficient space. There must be at least :space megabytes free.",
+    "suggest_run_update"        => "Commando niet beschikbaar. Voer alsjeblieft app:update eom te downloaden en te upgraden naar de nieuwste beschikbare versie : %s.",
+    "suggest_run_upgrade"       => "Er wordt al een upgrade uitgevoerd. Voer alsjeblieft app:upgrade.",
```

Comment:
```suggestion
    "suggest_run_upgrade" => "Er wordt al een upgrade uitgevoerd. Voer alsjeblieft app:upgrade uit.",
```

Not 100% sure about this sentence.